webapp/webpack.config.js

```diff
@@ -10,52 +42,37 @@ module.exports = {
         modules: [
             'src',
             'node_modules',
+            path.resolve(__dirname),
         ],
-        extensions: ['*', '.js', '.jsx'],
+        extensions: ['*', '.js', '.jsx', '.ts', '.tsx'],
     },
     module: {
         rules: [
             {
-                test: /\.(js|jsx)$/,
+                test: /\.(js|jsx|ts|tsx)$/,
                 exclude: /node_modules/,
                 use: {
                     loader: 'babel-loader',
                     options: {
                         cacheDirectory: true,
-                        plugins: [
-                            '@babel/plugin-proposal-class-properties',
-                            '@babel/plugin-syntax-dynamic-import',
-                            '@babel/proposal-object-rest-spread',
-                        ],
-                        presets: [ // Babel configuration is in .babelrc because jest requires it to be there.
```

Comment:
Interesting how the old code here said the config was in a separate file, but then it went on to define/duplicate the config in this file.